tests/qchem/test_tapering.py

```diff
@@ -629,3 +630,200 @@ def test_taper_obs(symbols, geometry, charge):
             @ scipy.sparse.coo_matrix(state_tapered).T
         ).toarray()
         assert np.isclose(obs_val, obs_val_tapered)
+
```

Comment:
Might consider a test for computing the ground state energy with the tapered excitation gates for a simple molecule.